### PR TITLE
LPS-188968 | Add test ManagerCanEditVeriifiedOrganization

### PR DESCRIPTION
**Pull request by bbqbronson**
https://liferay.atlassian.net/browse/LPS-188968

### CODE REVIEW

**Comment by bronsonsoda** [comment]
ci:test:osb-site-initializer-evp

**Comment by bronsonsoda** [comment]
ci:stop

**Comment by bronsonsoda** [comment]
ci:test:osb-site-initializer-evp

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 1]
```diff
@@ -61,6 +61,30 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-176607. Assert manager can edit a verified organization."
+	@priority = 5
+	test ManagerCanEditVeriifiedOrganization {
```
Comment (nikki-pru) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:66]: Please fix typo "Veriified" should be "Verified"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 2]
```diff
@@ -61,6 +61,30 @@ definition {
...
+		}
+
+		task ("When the manager goes to edit an organization with 'Verified' status") {
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("Then the manger is able to make changes") {
+			VPOrganization.kebabMenuOrganization(organizationName = "Organization 4");
+
+			AssertElementPresent(
```
Comment (nikki-pru) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:82]: This only checks that the Edit option is available. Please check that the manager is able to edit the organization.
1. Navigate into the organization
2. Make edits
3. Save edits
4. View organization that the edits are applied/persist.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 3]
```diff
@@ -61,6 +61,30 @@ definition {
...
+				userLoginFullName = "Manager User");
+
+			EVPUtils.gotoEVP();
+		}
+
+		task ("When the manager goes to edit an organization with 'Verified' status") {
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("Then the manger is able to make changes") {
```
Comment (nikki-pru) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:79]: Typo on 'manager'

**Review by nikki-pru — CHANGES_REQUESTED** [review]

**Comment by nikki-pru** [comment]
@bronsonsoda , please see requested changes above

**Comment by bronsonsoda** [comment]
How do i go about resolving conflicts when using an old commit to work from 🤔

**Comment by bronsonsoda** [comment]
Believe I did it

**Comment by joelrocha68** [comment]
This task is still in conflict

**Comment by kevenleone** [comment]
Archiving due inactivity

**Pull request closed without merging**